fabric-gui-v0/src/main/java/net/fabricmc/fabric/api/event/client/gui/ScreenInitCallback.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.fabricmc.fabric.api.event.client.player;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+import net.minecraft.client.gui.screen.Screen;
+import net.minecraft.client.gui.widget.AbstractButtonWidget;
+
+/**
+ * This event is emitted on the client to initialize and redraw gui screens
+ * whenever opened. Mods consuming this event are allowed to interact with
+ * the screen to add their own buttons and elements.
+ *
+ * Additional credit to Killjoy1121 for the initial implementation of this event.
+ */
+@FunctionalInterface
+public interface ScreenInitCallback {
```

Comment:
Formatting is all wrong, use the formater that should have been setup

Reply:
I don't see what's wrong here...

Reply:
We use tabs, everything here is spaces.

Reply:
:T Killjoy used to yell at me for using tabs.

Reply:
Still doesnt seem right, we have an intelij formatter that you can run. If you dont have intelij it can be done when merging, so dont worry about that too much now.